k2/python/k2/fsa_algo.py

```diff
@@ -36,6 +37,38 @@ def linear_fsa(symbols: Union[List[int], List[List[int]]]) -> Fsa:
     return fsa
 
 
+def linear_fst(symbols: Union[List[int], List[List[int]]],
+               aux_symbols: Union[List[int], List[List[int]]]) -> Fsa:
+    '''Construct an linear FST from symbols and it's corresponding
+       auxiliary symbols.
+
```

Comment:
Maybe better to add a Caution here to say that we don't support aux_labels as a ragged tensor for now. We can fix it later if we really find we need `linear_fst` to handle this case (i.e. aux_symbols is ragged tensor)

Reply:
I prefer to use `labels` instead of `symbols` here.  I would like to standardize on only using the word 'symbols' in the context of 'symbol table', and use 'labels' elsewhere.

Reply:
+2, can't agree any more.

Reply:
This is a historical issue. 

The C++ API has been using `symbols` and `linear_fsa` just follows the convention. Later on, `symbols` in Python
means symbol table but then argument name for `k2.linear_fsa` remains up to now.

Reply:
I will change `symbols` to `labels` also in `linear_fsa`.